frontend/src/styles/operator-editor-page.scss

```diff
@@ -567,7 +566,7 @@
 
   .oh-markdown-viewer__content {
     .CodeMirror, .CodeMirror-scroll {
-      min-height: initial;
+      min-height: initial;     
```

Comment:
nit: Please remove extra spaces. we should probably add stylelint at some point.